### DISCUSSION TITLE
Raman spectra from different laser energy

### DISCUSSION THREAD

**Discussion by rkarkee101** [nguyen-group/QERaman#18] [Q&A] [answered]
Hi, 

I am getting different Raman spectra on different laser energy. 
For example; I got following plot in 1.95 eV

![raman1](https://github.com/nguyen-group/QERaman/assets/76083361/23872ba9-0acb-44de-909b-91b01e37e437)

And then 2.33 eV shows:
![raman2](https://github.com/nguyen-group/QERaman/assets/76083361/cbaffd00-d344-4781-9a74-f90ef189c5f2)

There is a huge difference and I am confused which one is true. Can you please suggest me here?

Thanks
Best
Rijan

**Comment by nguyen-group** [comment, marked as answer]
Hi Rijan,

Thank you for your plotting.
For the resonance Raman spectra, the helicity-dependent Raman spectra should depend on the laser energy. To know which one is correct, we should compare it to the published experimental data or based on the group theory. 
You can find more information on "Deep-Ultraviolet and Helicity-Dependent Raman Spectroscopy for Carbon Nanotubes and 2D Materials" https://onlinelibrary.wiley.com/doi/abs/10.1002/smll.202308558

Best,
Nguyen
